### PR TITLE
Add load_frozen_state also to LoadStateFromFile gRPC requests.

### PR DESCRIPTION
**Pull request by MichaelGrupp**

### CODE REVIEW

File: cartographer/mapping/map_builder_interface.h [hunk 1]
```diff
@@ -92,7 +92,7 @@ class MapBuilderInterface {
   // Loads the SLAM state froma a pbstream file. Returns the remapping of new
```
Comment (gaschler) [inline cartographer/mapping/map_builder_interface.h]: froma -> from

File: cartographer/mapping/map_builder_interface.h [hunk 2]
```diff
@@ -92,7 +92,7 @@ class MapBuilderInterface {
...
   // trajectory_ids.
   virtual std::map<int /* trajectory id in proto */, int /* trajectory id */>
-  LoadStateFromFile(const std::string& filename) = 0;
+  LoadStateFromFile(const std::string& filename, bool load_frozen_state) = 0;
```
Comment (gaschler) [inline cartographer/mapping/map_builder_interface.h]: Note: changes the public API, but becomes consistent with LoadState

Comment (MichaelGrupp) [reply cartographer/mapping/map_builder_interface.h]: That's why I changed it

**Comment by gaschler** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user gaschler. Merge queue now has a length of 1.

**Pull request merged**